README.md

```diff
@@ -14,9 +14,9 @@ For more information, please visit [https://www.freeclimb.com/support/](https://
 ## Installing
 
 ```sh
-npm install @freeclimb/sdk@3.0.1
+npm install @freeclimb/sdk@3.0.3
 or 
-yarn add @freeclimb/sdk@3.0.1
+yarn add @freeclimb/sdk@3.0.3
```

Comment:
@rchi-vail why the extra patch bump? currently what is released is 3.0.1 - next version should be 3.0.2.

Reply:
@ngarg-vail good question.  I initially thought it should be 3.0.3 because 3.0.2 is in [openapitools.json](https://gitlab.vailsys.com/vail-cloud-services/openapi-spec/-/commit/3d5659e86a44c3077bedb26ae197af4f147a3443 ), but I think you're right.  If 3.0.1 is what's out there, it should be 3.0.2, which means that I made a mistake when bumping to 3.0.2 on openapi spec.  Long story short, if I bump this to 3.0.2 and leave openapitools as-is, we should be good